frontend/packages/console-app/src/components/cloud-shell/__tests__/CloudShellTerminal.spec.tsx

```diff
@@ -14,7 +12,7 @@ jest.mock('@console/internal/components/utils/k8s-watch-hook', () => ({
 describe('CloudShellTerminal', () => {
   it('should display loading box', () => {
     (useK8sWatchResource as jest.Mock).mockReturnValueOnce([null, false]);
-    const wrapper = shallow(<InternalCloudShellTerminal username="user" />);
+    const wrapper = mount(<InternalCloudShellTerminal username="user" />);
```

Comment:
We want to use `shallow` rendering. Checking for `LoadingBox` is no longer correct. This test needs updating.